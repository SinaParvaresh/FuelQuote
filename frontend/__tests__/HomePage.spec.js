// // const filterByTerm = require("../src/filterByTerm");


// // describe("Profile Management", () => {

// //     test("it should handle valid login info", () => { 

// //     });

// //     test("it should handle invalid login info", () => { 

// //     });

// //   });

// //   <ProfileManagement />



// import React from 'react';
// import renderer from 'react-test-renderer';

// import ProfileManagement from 'C:/Users/Ankur/Desktop/Computer Science/COSC 4353 - Singh/Fuel Quote Project/Project_Files/frontend/fuelquote/src/components/pages/profileManagement';
// // import { Counter } from './App';

// describe('Profile Management', () => {
//   test('snapshot renders', () => {
//     const component = renderer.create(<ProfileManagement />);
//     let snap = component.toJSON();
//     expect(snap).toMatchSnapshot();
//   });
// });





import React from 'react';
import {act, create} from 'react-test-renderer';
import { MemoryRouter } from 'react-router-dom';

import HomePage from '../src/components/pages/homePage';
// import { Counter } from './App';

describe('Home page', () => {
  test('if render matches snapshot', async() => {
    let component;
    await act(async() => { 
      component = create(<MemoryRouter><HomePage /></MemoryRouter>);
    });
    let snap = component.toJSON();
    expect(snap).toMatchSnapshot();
  });
});