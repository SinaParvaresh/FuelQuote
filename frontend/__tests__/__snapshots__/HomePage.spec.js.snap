// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home page if render matches snapshot 1`] = `
<div
  className="homePage-background"
>
  <div
    className="HomePage"
  >
    <h1
      className="text-uppercase font-weight-bold text-lg-center"
    >
      Welcome
      <br />
      to the Fuel Quote Website
      <br />
      of GROUP 33!!!
    </h1>
    <br />
    <a
      href="/login"
      onClick={[Function]}
      style={
        Object {
          "textDecoration": "none",
        }
      }
    >
      <button
        className="btn btn-outline-secondary btn-lg btn-block text-uppercase font-weight-bold"
      >
        Click here to Login!
      </button>
    </a>
  </div>
</div>
`;
